cpp/ql/lib/semmlecode.cpp.dbscheme.stats

```diff
@@ -336,6 +336,18 @@
             <k>@scalable_vector_count</k>
             <v>124</v>
         </e>
+        <e>
+            <k>@complex_fp16</k>
+            <v>124</v>
+        </e>
+        <e>
+            <k>@complex_std_bfloat16</k>
+            <v>124</v>
+        </e>
+        <e>
+            <k>@complex_std_float16</k>
+            <v>124</v>
+        </e>
```

Comment:
Note that I just hacked this in, as stats file generation is currently broken. I do a proper update once that is fixed.